octoprint_mrbeam/materials.py

```diff
@@ -38,7 +39,7 @@ def get_custom_materials(self):
 
     def put_custom_material(self, key, material):
```

Comment:
So this is also used when loading older custom materials from a file?

Reply:
No this is when you add new materials, import JSON file to restore materials, and when the migration script sanitize the already existing materias